sui_core/Cargo.toml

```diff
@@ -31,6 +31,11 @@ clap = { version = "3.1.14", features = ["derive"] }
 bincode = "1.3.3"
 fdlimit = "0.2.1"
 schemars = "0.8.8"
+curve25519-dalek = { version = "3", default-features = false }
+structopt = "0.3.26"
+log = "0.4.14"
```

Comment:
I think we use `tracing` rather than `log`:
```
tracing = { version = "0.1.34", features = ["log"] }
```

Reply:
Thanks! removed